documentation/local_quickstart.rst

```diff
@@ -0,0 +1,179 @@
+***********************************
+Local Development Quick Start
+***********************************
+
+The Python Component System (PCS) can also be used within a project you're
+developing in your local development environment.  
+
+In this quickstart, we'll work through an example of using PCS in a local
+project.  We'll start by installing ``agentos``. Run the following in your
+shell::
+
+    pip install agentos
+
+Now, let's create a new directory for our project and initialize it.  Run the
+following in your shell::
+
+    mkdir quickstart
```

Comment:
Nit: it's a aesthetic choice, but I find it a little cleaner to let the command create the dir:
```
agentos init quickstart
cd quickstart
```

Reply:
I dig!